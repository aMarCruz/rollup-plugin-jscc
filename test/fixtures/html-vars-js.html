<!DOCTYPE html>
<!--
  App main template (_TITLE is an external memvar)
  @license MIT
-->
//#set _SUFFIX 12345
<html lang="en">
<head>
  <meta charset="UTF-8">
  <title>$_TITLE</title>
</head>
<body>
  <!--
    all the jscc comments will be removed, so you can use them in html
  -->
  <h1>$_TITLE</h1>
  <!-- #if -->
  //#if _DEBUG
  <script src="js/debug.js?$_SUFFIX"></script>
  //#else
  <script src="js/release.js?$_SUFFIX"></script>
  //#endif
  <script src="js/myapp.js?$_SUFFIX"></script>
</body>
</html>
