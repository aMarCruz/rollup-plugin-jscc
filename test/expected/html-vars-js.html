<!DOCTYPE html>
<!--
  App main template (_TITLE is an external memvar)
  @license MIT
-->
<html lang="en">
<head>
  <meta charset="UTF-8">
  <title>My App</title>
</head>
<body>
  <!--
    all the jscc comments will be removed, so you can use them in html
  -->
  <h1>My App</h1>
  <!-- #if -->
  <script src="js/release.js?12345"></script>
  <script src="js/myapp.js?12345"></script>
</body>
</html>
